src/util/containers/ContainerManager.ts

```diff
@@ -0,0 +1,27 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+
+/**
+ * Handles everything related to container identifiers and how they relate to other identifiers.
+ */
+export interface ContainerManager {
+  /**
+   * Verifies if this identifier could be valid.
+   * This does not check if this identifier actually exists,
+   * but checks if the identifier is something that could be a valid identifier.
+   */
+  isValidIdentifier: (identifier: ResourceIdentifier) => boolean;
```

Comment:
```suggestion
  supportsIdentifier: (identifier: ResourceIdentifier) => boolean;
```

(I find _valid_ weird, given that it is validity in a specific context.)